components/app-configs/mlde/workspace_config/mlde-task-template.yaml

```diff
@@ -0,0 +1,16 @@
+environment:
+  environment_variables: {}
+  image: nolanc/pyspark-nb:spark3.5-rapids1.12-azure3.4-autosparksession0.0.2
+  pod_spec:
+    spec:
+      serviceAccountName: mlde-worker
+      containers:
+      - name: determined-container
```

Comment:
`determined-container` is the runtime container for MLDE. There is also `determined-init-container`, which generally does not need to be touched. However, if we want to do things like inject env variables from secrets, this is where we want to do it.